open_spiel/games/maedn.h

```diff
@@ -0,0 +1,281 @@
+// Copyright 2022 Thorsten Jungblut
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef OPEN_SPIEL_GAMES_MAEDN_H_
+#define OPEN_SPIEL_GAMES_MAEDN_H_
+
+#include <array>
+#include <memory>
+#include <set>
+#include <string>
+#include <vector>
+
+#include "open_spiel/spiel.h"
+
+// An implementation of Mensch-Aergere-Dich-Nicht (see
+// https://de.wikipedia.org/wiki/Mensch_%C3%A4rgere_Dich_nicht)
+//
+// Rules used:
+// - start field must be cleared as soon as possible
+// - throwing out own pieces is not possible
+// - only one dice roll even if no move is possible except if dice roll was
+//   a six, in this case, same player may roll again
+// - pieces may jump over each other on four final fields
+//
+// Parameters:
+//   none yet
+
+namespace open_spiel {
+namespace maedn {
+
+inline constexpr const int kNumPlayers = 4;
```

Comment:
I think this ought to be renamed `kMaxNumPlayers` to eliminate any possible confusion with the actual number of players.

Reply:
Good suggestion, it was indeed confusing. I renamed it as you suggested.